april_vision/cli/tools/list_cameras.py

```diff
@@ -0,0 +1,30 @@
+import argparse
+import logging
+
+from april_vision.calibrations import calibrations
+from april_vision.detect_cameras import find_cameras
+
+LOGGER = logging.getLogger(__name__)
+
+
+def main(args: argparse.Namespace) -> None:
+    cameras = find_cameras(calibrations, include_uncalibrated=True)
+    for camera in cameras:
+        LOGGER.info(camera)
```

Comment:
Can we format this a bit nicer with fixed width columns for index, vid/pid, name and calibration file.